modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -28,6 +28,7 @@ class DisabilityCompensationController < ClaimsApi::V1::Forms::Base
         skip_before_action :validate_json_format, only: %i[upload_supporting_documents]
         before_action :verify_power_of_attorney!, if: :header_request?
         skip_before_action :validate_veteran_identifiers, only: %i[submit_form_526 validate_form_526]
+        before_action :edipi_check, only: %i[submit_form_526 upload_form_526]
```

Comment:
what's the concept for not adding it to `validate_form_526` as well?

Reply:
My thought was because that doesn't submit anything, but I was only thinking of the error handling and not the bigger picture on that of what it is doing.  But now that you mention it, I guess we would still want to know this information during validation as well so I will get that added to the 2 validation endpoints

Reply:
@FonzMP Would we want this EDIPI check before the generate_pdf endpoint as well?  I guess my question is, would we want to give the ability to generate this PDF if they do not have this value?  Not sure if that matters or not.

Reply:
that's actually a relevant question - @klwatanabe , maybe this is a question for you? We can give people the ability to generate a PDF, but _should_ we give them the ability to without an EDIPI? If they submit using a generated PDF from our service, but try to establish a Claim in some other way, shape or form, will they ever succeed without said EDIPI? Assuming there's some way to do so without an EDIPI, but ? 

I don't have an answer for this Rockwell. I would assume we leave it off of generate_PDF, but that could cause hiccups?

Reply:
For our 526 auto-establishment process, we should do the upfront validation to confirm they have an edipi, so the we do not end up with a claim that can never be established.

Our generatePDF endpoint has minimum validation. We do not want to add this validation there. This endpoint is used to create a PDF to establish manually for backup submissions. This scenario would be an example of when the backup submission path would need to be used.